lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -9,44 +9,36 @@
 from six import text_type
 %>
 
-<header>
-  % if static.get_value('course_about_show_social_links', True):
-  <div class="social-sharing">
-    <div class="sharing-message">${_("Share with friends and family!")}</div>
-    ## TODO: this should probably be an overrideable block,
-    ##       or something allowing themes to do whatever they
-    ##       want here (and on this whole page, really).
-      <%
+    <header>
+        % if static.get_value('course_about_show_social_links', True):
+        <div class="social-sharing">
+            <div class="sharing-message">${_("Share with friends and family!")}</div>
+            ## TODO: this should probably be an overrideable block, ## or something allowing themes to do whatever they ## want here (and on this whole page, really).
+            <%
         site_domain = static.get_value('site_domain', settings.SITE_NAME)
         site_protocol = 'https' if settings.HTTPS == 'on' else 'http'
         platform_name = static.get_platform_name()
+        course_path = reverse('about_course', args=[text_type(course.id)])
+        course_url = f"{site_protocol}://{site_domain}{course_path}"
 
         ## Translators: This text will be automatically posted to the student's
         ## Twitter account. {url} should appear at the end of the text.
-        tweet_text = _("I just enrolled in {number} {title} through {account}: {url}").format(
+        tweet_text = _("I just enrolled in {number} {title} through {account} {url}").format(
```

Comment:
Why this removal?

Reply:
I removed because its confusing or lets say unusual to have the ':' char after mentioning an account on twitter. 
Before applying this change the generated would like:
`I just enrolled ..etc through @platformtwitteraccount: https..etc` and after it's `I just enrolled ..etc through @platformtwitteraccount https..etc`

Reply:
@rgraber is it okay now to merge, or do you still have notes? Since nutmeg is going to be released in a couple of days